AzureDay.WebApp/AzureDay.WebApp.Notification/Email/Template/RestorePassword.cs

```diff
@@ -0,0 +1,339 @@
+﻿// ------------------------------------------------------------------------------
```

Comment:
the whole file is redundant and should be removed, restore password will be handled by azure ad b2c